examples/twittermap/web/app/actor/TwitterMapPigeon.scala

```diff
@@ -0,0 +1,43 @@
+package actor
+
+import java.net.URI
+
+import akka.actor._
+import akka.stream.Materializer
+import org.eclipse.jetty.websocket.client.{ClientUpgradeRequest, WebSocketClient}
+import play.api.libs.json.JsValue
+import socket.TwitterMapClientSocket
+
+import scala.concurrent.ExecutionContext
+
+class TwitterMapPigeon (val cloudberryWS: String,
+                        val out: ActorRef)
+                       (implicit ec: ExecutionContext, implicit val materializer: Materializer) extends Actor with ActorLogging {
+
+  val twitterMapWebSocketClient: WebSocketClient = new WebSocketClient
+  val twitterMapClientSocket: TwitterMapClientSocket = new TwitterMapClientSocket(out)
```

Comment:
the names of these two variables are too similar. I couldn't figure out what does each mean. can you elaborate?